tools/sync_users/sync_users.py

```diff
@@ -122,20 +122,18 @@ def main():
     parser.add_argument('-f', '--file', required=True)
     parser.add_argument('-t', '--token', required=True)
     args = parser.parse_args()
-    # file_path='/Users/mrinali/projects/safetyculture-sdk-python/iauditor_users.csv'
-    file_path = args.file
+    # input_filepath='/Users/mrinali/projects/safetyculture-sdk-python/iauditor_users.csv'
+    input_filepath = args.file
     api_token = args.token
 
     sc_client = sp.SafetyCulture(api_token)
 
-    json_all_groups = json.loads(sc_client.get_all_groups_in_org().content)
+    all_group_details = json.loads(sc_client.get_all_groups_in_org().content)
     server_users = export_users.get_all_users()
 
-    process_desired_state(server_users, file_path)
-    process_server_state(server_users, file_path)
-    execute_actions(json_all_groups, sc_client)
-
-
+    process_desired_state(server_users, input_filepath)
+    process_server_state(server_users, input_filepath)
+    execute_actions(all_group_details, sc_client)
 
 if __name__ == '__main__':
     main()
```

Comment:
Calling a function main() is not very intuitive. Consider renaming to something like sync_users()